pkg/environment/environment_loader.go

```diff
@@ -19,6 +19,7 @@ package environment
 import (
 	"errors"
 	"fmt"
+	"github.com/dynatrace-oss/dynatrace-monitoring-as-code/pkg/manifest"
```

Comment:
This now depends on Manifests where it didn't before. 
Not sure if that may come to hurt us by becoming a cyclic dependency at some point. 
Maybe you could pull the Filter into it's own util to avoid that completely?